app.js

```diff
@@ -3,59 +3,95 @@ console.log("userName is " + userName);//displays in the developer console what
 var score = 0;//Creates the score variable that equals 0.
 console.log("Current score is : " + score);//displays current score.
 
-var answerOne = prompt(userName + ", am I from Washington?");//creates the variable which will prompt the question and store the user input.
-console.log("answerOne is " + answerOne);//displays in the developer console what the user input was.
-if (answerOne.toLowerCase() === "yes") {//compares the user input in lowercase to the string yes.
-  alert("Great job " + userName + "!");//If the above statement is true this string will be displayed.
-  score++;//Adds 1 to the score for getting the correct answer.
-  console.log ("Current score is : " + score);//displays current score in the console.
-} else if (answerOne.toLowerCase() === "no") {//compares the user input, converted to lower case characters, to the string no.
-  alert("Sorry " + userName + " that is incorrect.");//If the above statement is true this string will be displayed.
-} else {//If the user input not 'yes' or 'no' this will run.
-  alert("I'm sorry. The only accepted answers are 'yes' and 'no'.");//will be displayed when the above is true.
-}
+var els = [
+  document.getElementById("qOne"),
+  document.getElementById("qTwo"),
+  document.getElementById("qThree"),
+  document.getElementById("qFour"),
+  document.getElementById("qFive"),
+  document.getElementById("numCorr")
+];
 
-var answerTwo = prompt(userName + ", did I work at Sega?");//creates the variable answerTwo which prompts the question and stores the answer in yesNoAnswerTwo.
-console.log("answerTwo is " + answerTwo);//displays the value of answerTwo in the console.
-if (answerTwo.toLowerCase() === "no") {//compares the user input to no
-  alert("That is correct " + userName + ".");//if the above statement is true this string will display.
-  score++;//Adds 1 to the score for getting the correct answer.
-  console.log ("Current score is : " + score);//displays the current score in the console.
-} else if (answerTwo.toLowerCase() === "yes") {//compares the user input to yes.
-  alert("I did not work at sega.");//if the above statement is true this string will display.
-} else {//if the user input is not yes or no this will run.
-  alert("Only 'yes' and 'no' are accepted for this question.");//displays when the above is ture.
-}
+var question = [
```

Comment:
Since this array holds multiple questions, you may want to name it as plural (questions), just to stay consistent. Your elements variable is plural (els)